src/Microsoft.Framework.Runtime/ProjectResolver.cs

```diff
@@ -30,25 +37,19 @@ public bool TryResolveProject(string name, out Project project)
         {
             project = null;
 
-            foreach (var searchPath in _searchPaths)
+            ProjectInformation projectInfo;
+            if (_projects.TryGetValue(name, out projectInfo))
             {
-                var projectPath = Path.Combine(searchPath, name);
-
-                if (Project.TryGetProject(projectPath, out project))
-                {
-                    return true;
-                }
+                project = projectInfo.Project;
+                return true;
```

Comment:
Should be
```c#
return project != null;
```
`_projects` contains potential project dirs that can be non-project dirs.